dashboard/src/components/skills/EditSkill.vue

```diff
@@ -178,10 +173,32 @@ limitations under the License.
             </div>
           </div>
 
-            <hr class="my-0 pb-1"/>
-            <hr class="mt-0 pt-0"/>
+            <hr class="mt-0"/>
 
-          <div class="">
+            <div class="row mb-2">
+              <div class="col-12 col-lg-auto">
+                  <b-form-checkbox data-cy="selfReportEnableCheckbox" id="self-report-checkbox"
+                                   class="d-inline" v-model="selfReport.enabled"
+                                   v-on:input="updatedSelfReportingStatus"/>
+                Self Reporting <inline-help msg="Check to enable ability for users to self report this skill."/>:
```

Comment:
msg might be better phrased as "Check to enable self-reporting of this skill by users"

Reply:
updated